testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -2074,11 +2074,20 @@ def test_frame_interval_convention():
         universe1 = MDAnalysis.Universe(PSF, DCD)
         array1 = universe1.trajectory.timeseries(skip=10)
         universe2 = MDAnalysis.Universe(PSF, DCD, in_memory=True,
-                                        in_memory_frame_interval=10)
+                                        in_memory_step=10)
         array2 = universe2.trajectory.timeseries()
         assert_equal(array1, array2,
                      err_msg="Unexpected differences between arrays.")
 
+    @staticmethod
+    def test_slicing():
+        universe = MDAnalysis.Universe(PDB_small, DCD)
+        # Skip only the last frame
+        universe.transfer_to_memory(start=0, stop=-1)
```

Comment:
`start` `stop` and `step` are values for a python slice ie
```python
a = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

a[start:stop:step]

```
So `start=0` and `stop=-1` is just everything, whereas `start=2` `stop=8` should give a known length.

The 97 in that test is the length of the entire iterable, so if you play with start/stop/step you should start seeing it changing exactly like slicing a list would work